src/config.py

```diff
@@ -51,6 +51,14 @@
     TEST_PODCAST_DB_NAME
 )
 
+# Facebook API information
+FACEBOOK_APP_ID = os.environ.get('FACEBOOK_APP_ID')
+FACEBOOK_APP_SECRET = os.environ.get('FACEBOOK_APP_SECRET')
```

Comment:
Maybe mention what we should be assigning these two in the README if we are trying to run on local?